resources/production/chainspec.toml

```diff
@@ -44,6 +44,18 @@ start_protocol_version_with_strict_finality_signatures_required = '1.5.0'
 # in a protocol version before
 # `start_protocol_version_with_strict_finality_signatures_required`.
 legacy_required_finality = 'Strict'
+
+# legacy_migration_handling = ENUM { Lazy, Account, AccountContract }
```

Comment:
this line should be delete, since we went w/ 2 bools vs 1 enum

Reply:
Will do!

Reply:
Removed